trade-finance-manager-ui/server/controllers/case/amendments/amendCoverEndDate.test.js

```diff
@@ -1,8 +1,9 @@
-import { isTfmFacilityEndDateFeatureFlagEnabled, TEAM_IDS } from '@ukef/dtfs2-common';
+import { FACILITY_TYPE, isTfmFacilityEndDateFeatureFlagEnabled, TEAM_IDS } from '@ukef/dtfs2-common';
```

Comment:
I dont think you want to delete the `MAPPED_FACILITY_TYPE` entirely, my comment was referring to specifically the Loan value. CASH and CONTINGENT are actually different because of the weird mapping in tfm-api

Reply:
might have commented this twice because git being weird oops

Reply:
I've added mapped facility type to libs common (with a slightly diff name of `facility_type_mapped` in the exploreer to group with with `facility type`). That was an adventure of old code